test/sparse/sparsevector.jl

```diff
@@ -184,6 +184,29 @@ let x = sprand(100, 0.5)
     @test Array(r) == Array(x)[I]
 end
 
+# boolean array index
+let x = sprand(10, 10, 0.5)
+    I = rand(1:size(x, 2), 10)
+    bI = falses(size(x, 2))
+    bI[I] = true
+    @which x[1,bI]
```

Comment:
ah right I didn't see the `@which` call a few lines above - either check something about the result (comes from the right module maybe?) or get rid of it?